### PR TITLE
tune2fs: add page

### PR DESCRIPTION
**Pull request by samuel-w**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

Relates to #4250

### CODE REVIEW

File: pages/linux/tune2fs.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# tune2fs
+
+> Adjust paramters of an ext2, ext3 or ext4 file system.
```
Comment (sbrl) [inline pages/linux/tune2fs.md:3]: Can this be done while the filesystem is mounted, or does it have to be unmounted first?

Comment (samuel-w) [reply pages/linux/tune2fs.md:3]: They all can be done live I believe. I checked all the ones in the tldr.

File: pages/linux/tune2fs.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# tune2fs
+
+> Adjust paramters of an ext2, ext3 or ext4 filesystem.
```
Comment (sbrl) [inline pages/linux/tune2fs.md:3]: Nice! Sorry about the late reply. What about this then?
```suggestion
> Adjust parameters of an ext2, ext3 or ext4 filesystem.
> May be used on mounted filesystems.
```

**Review by sbrl — COMMENTED** [review]
Thanks, @samuel-w :smiley_cat:

**Review by sbrl — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]
Interesting! LGTM. :+1:

**Pull request merged**